draft-bft-rats-kat.md

```diff
@@ -304,6 +304,12 @@ of the used KAK and therefore the linkage between the two layers.
 
 # Security Considerations
 
+## Semantics of Key Attestation
+
+The semantics of the KAT, i.e. what exact properties are attested, is highly implementation dependent. At the very least, the platform MUST NOT sign the KAT unless the IK's private key is generated by the Trusted Computing Base and never leaves its control.
```

Comment:
> At the very least, the platform MUST NOT sign the KAT unless the IK's private key is generated by the Trusted Computing Base and never leaves its control.

I'm not sure, another option would be to make the properties of the key (e.g., whether it can be exported) explicit in the KAT.

Reply:
@ionut-arm I was assuming this is an ephemeral key, and therefore there is no motivation to export it and so we can assume it is protected from export. Are you aware of implementations where this is not the case?

Reply:
I think importing/whether the key was imported was what I actually had in mind. I remember some conversations around the idea that long-lived keys that had (similarly long-lived) x509 certs associated with them might be migrated to an environment that offered attestation, and might thus benefit from both PKIX and RA.

But no, I wouldn't say I could name or identify a system where this is being done, so feel free to ignore!

Reply:
I would leave the door open for an encrypted key export since some hardware security technologies can only store a limited number of keys and those key storage slots are typically reserved for persistent keys. Hence, for short-lived keys the pattern is that the encrypted key is stored somewhere on the device and can only be decrypted by the hardware security module. This use case should be fine as well. 

Here is the suggested text: 

```suggestion
The semantics of the KAT, i.e. what exact properties are attested, is highly implementation dependent. At the very least, the platform MUST NOT sign the KAT unless the IK's private key is generated by the Trusted Computing Base and is never exported in the clear.
```

Reply:
please give me until the end of the week to provide some feedback here (abroad the next days)

Reply:
> please give me until the end of the week to provide some feedback here (abroad the next days)

@henkbirkholz bumping this back to the top of your inbox.